Stonks/Forms/FormSmartSaver.cs

```diff
@@ -35,100 +39,17 @@ public void setTrackBarValues()
             var utilitiesExpense = Program.financialPlan.GetExpense(ExpenseType.Utilities);
             var otherExpense = Program.financialPlan.GetExpense(ExpenseType.Other);
             var savings = Program.financialPlan.Savings;
-            var max = Program.financialPlan.GetMaxExpense(); 
-
-            //Setting the values to represent the track bars
-            if (housingExpense != null) {
-                trackBarHousing.Maximum = Convert.ToInt32(max);
-                trackBarHousing.Value = Convert.ToInt32(housingExpense.PlannedValue); //Fix if value becomes higher than expense(just like savings?)
-                labelHousingExpense.Text = (housingExpense.PlannedValue).ToString("€#.#");
-                labelHousingExpensesActual.Text = (housingExpense.Value).ToString("€#.#");
```

Comment:
I see you've been working on the old file. It's not a big deal this time, but have in mind always pulling before working.